### PR TITLE
Cleaning + Adaptive links

### LINKED ISSUE #3727: M. Accion
**_Channel Name:_** M. Accion
**_Broken Link (from playlist):_** Spain
**_Notes (optional):_** All Movistar channels are down. Seems to be related to #3707

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Closes #3727 by deleting links that do not work. However, the other Movistar Astra streams still work.

### CODE REVIEW

**Comment by Dum4G** [comment]
Same here, we've got automatic cleaning branch ready. Cleanup can be continued from there

File: channels/cn.m3u [hunk 1]
```diff
@@ -1,14 +1,12 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="ATVAsiaYaZhouDianShiTai.cn" tvg-name="ATV Asia (亞洲電視台)" tvg-country="CN" tvg-language="Chinese" tvg-logo="https://www.hkatv.com/images/logo.png" group-title="",ATV Asia (亞洲電視台)
-https://15813114727637862976168173814320.live.prod.hkatv.com/a1_cbr_hi_1080p.m3u8
-#EXTINF:-1 tvg-id="BlueMeiJuPinDao.cn" tvg-name="Blue 美剧频道" tvg-country="CN" tvg-language="Chinese" tvg-logo="https://parco-zh.github.io/demo/BLUE.png" group-title="",Blue 美剧频道
-http://210.210.155.35:80/session/676c25ba-724e-11e9-834b-8ac28bf32f5a/dr9445/h/h16/02.m3u8
```
Comment (Dum4G) [inline channels/cn.m3u:5]: works

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/cn.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/cn.m3u:5]: Geo-locked then : 
image

**Comment by Dum4G** [comment]
Is it time to reimplement it?

**Comment by LaneSh4d0w** [comment]
Give me a few moments to re-examine the Chinese playlist and then we're good to go.

**Comment by spotanjo3** [comment]
Portugal Channel.. Can't find the right place for this because those are double links which is not need. 

Kuriakos TV (1080p) (Please remove it) One of it is already in the list is (576p)
Bloomberg TV Europe (720p) (Please remove it) 2160p is already in the list.
Bloomberg TV EMEA Live EVENT (720p) (Please remove it and find one) It is not 720. A little blurry.
Kuriakos Cine (1080p) (Please remove it) One of it is already in the list
TV Fátime (1080p) (Please removed it)
TV Fátima (1080p) (Please removed it) Both are the same
Sobrenatural TV (720) (Not 24/7) (Please removed it) One of it is already (1080p) in the list
RPT1 [not 24/7] (Please removed it) Already have RPT1

**Comment by LaneSh4d0w** [comment]
This will be good for now. Another batch of cleans and adaptive links will come on due time.

File: channels/cn.m3u [hunk 2]
```diff
@@ -37,137 +37,121 @@ http://223.110.243.171/PLTV/3/224/3221227204/index.m3u8
...
-#EXTINF:-1 tvg-id="18" tvg-name="CGTN" tvg-country="CN" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/CGTN.svg/800px-CGTN.svg.png" group-title="News",CGTN (720p)
+#EXTINF:-1 tvg-id="CGTNEnglish.cn" tvg-name="CGTN English" tvg-country="CN" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/CGTN.svg/800px-CGTN.svg.png" group-title="News",CGTN English (720p)
 https://news.cgtn.com/resource/live/english/cgtn-news.m3u8
-#EXTINF:-1 tvg-id="CGTN.cn" tvg-name="CGTN" tvg-country="CN" tvg-language="" tvg-logo="" group-title="",CGTN (720p)
+#EXTINF:-1 tvg-id="CGTNEspanol.cn" tvg-name="CGTN Español" tvg-country="CN" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/CGTN.svg/800px-CGTN.svg.png" group-title="News",CGTN Español (720p)
 https://news.cgtn.com/resource/live/espanol/cgtn-e.m3u8
-#EXTINF:-1 tvg-id="CGTN1.cn" tvg-name="CGTN 1" tvg-country="CN" tvg-language="Spanish" tvg-logo="https://astrocontent.s3.amazonaws.com/Images/ChannelLogo/Pos/509_300.png" group-title="",CGTN 1
-https://livees.cgtn.com/1000e/prog_index.m3u8
```
Comment (Dum4G) [inline channels/cn.m3u:143]: whats up with those prog_index links? All work for me

Comment (LaneSh4d0w) [reply channels/cn.m3u:143]: If you open the m3u8 file above, those links are in there.

image

**Review by Dum4G — APPROVED** [review]

**Pull request merged**